### ISSUE TITLE
Show the support for the latest Outlook version

### ISSUE THREAD

**Issue by webdevelukas** [hteumeuleu/caniemail#368]
The new Outlook for Windows is out and it seems like it supports a lot more html/css features: https://techcommunity.microsoft.com/t5/outlook-blog/new-outlook-for-windows-now-available/ba-p/3932068

I recently created a HTML signature for our company. And while it didn't look so great on the old outlook version, it now looks as great as on the other email clients. 😄

**Comment by M-J-Robbins** [comment]
Adding to this, the latest Outlook on Mac seems to have moved backwards in support, which is a shame. 

But the good news is, it's now more in line with the new Windows PWA version as well as the webmail version.  I think it would be good to retest all 3 of these to see how close they are aligned and where there issues are that we need to look out for.

**Comment by hteumeuleu** [comment]
Yes, this is a very good point. I started an `outlook-one` branch early this year but stopped halfway through it. I'll see if I can do both this and the mac version to update the site later this month or early next month.

**Comment by Yin117** [comment]
How's the progress on this, I didn't realise the latest version of Outlook (2021) was newer than 2019, and had been basing some of my developments on this.
For example, `width="100"` appears to work fine for `<img/>` tags.
